server/notebooks/serializers.py

```diff
@@ -62,4 +62,7 @@ def validate(self, attrs):
     class Meta:
         model = NotebookRevision
         fields = ("id", "title", "created", "content")
-        write_only_fields = "notebook_id"
+        extra_kwargs = {
+            "notebook_id": {"write_only": True},
+            "is_draft": {"write_only": True},
```

Comment:
I think we do in fact want to include the "is_draft" field in the serialized representation here. I think `is_draft` a read-only field:

https://www.django-rest-framework.org/api-guide/fields/#read_only

You can set the `is_draft` property when creating the revision around here:

https://github.com/iodide-project/iodide/blob/362eb451675b46268cc36ee7a34feb2139d3c630/server/notebooks/api_views.py#L149